dataprofiler/profilers/profile_builder.py

```diff
@@ -834,6 +859,29 @@ def report(self, report_options: dict = None) -> dict:
         """
         raise NotImplementedError()
 
+    @classmethod
+    def load_from_dict(cls, data) -> BaseProfiler:
+        """
+        Parse attribute from json dictionary into self.
+
+        :param data: dictionary with attributes and values.
+        :type data: dict[string, Any]
+
+        :return: Profiler with attributes populated.
+        :rtype: BaseCompiler
+        """
+        profiler = cls(None)
+
+        for attr, value in data.items():
+            if "times" in attr:
+                setattr(profiler, "times", defaultdict(float, value))
+            if "_profiles" in attr:
+                value = load_compiler(value)
```

Comment:
This is diff depending on struct or unstruct, right?
struct would be the `structuredcolprofiler` I believe.

maybe we can update `_profiles` before entering here and allow it to just be set.